.github/workflows/docs_test.yml

```diff
@@ -24,7 +24,7 @@ env:
 jobs:
   test-python:
     name: Test doc python code
-    runs-on: "warp-ubuntu-latest-x64-4x"
+    runs-on: ubuntu-24.04
```

Comment:
maybe use larger worker?

Reply:
Tried 8x, still the same OOM.  Is 16x too overkill? :)

Reply:
Ubuntu-24.04 is 4 core for OSS projects. Do we need 8x/16x machine?